config/routes.rb

```diff
@@ -9,4 +9,5 @@
   delete "/logout", to: "sessions#destroy"
   root "static_pages#home"
   resources :users
+  resources :account_activations, only: [:edit]
```

Comment:
dùng ruby percent notation (tìm hiểu cái này nha)
```suggestion
  resources :account_activations, only: %i(edit)
```

Reply:
ok anh